### PR TITLE
Cap requirements for network integration tests

### PR DESCRIPTION
**Pull request by danielmellado**
This commit adds a version for every requirement on the network
integration tests. On top of that it's also adding a constraints file
in order to avoid getting over cap per release.

Signed-off-by: Daniel Mellado <dmellado@redhat.com>

### CODE REVIEW

**Comment by danielmellado** [comment]
Shippable CI Issue, failure is not related, reopening to retrigger.

File: test/integration/network-integration.upper-constraints.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pexpect===4.7.0
```
Comment (mattclay) [inline test/integration/network-integration.upper-constraints.txt:1]: What is using this file?

File: test/integration/network-integration.requirements.txt [hunk 2]
```diff
@@ -1,5 +1,4 @@
-pexpect # for _user test
-scp # for Cisco ios
-selectors2 # for ncclient
-ncclient # for Junos
-jxmlease # for Junos
+pexpect>=4.7.0 # ISC license
```
Comment (mattclay) [inline test/integration/network-integration.requirements.txt:1]: Why remove the comments indicating what is using each requirement?

**Comment by mattclay** [comment]
Closing since the network tests are no longer in this repo.

**Pull request closed without merging**